azext_iot/tests/iothub/devices/test_hub_control_plane_export_import_int.py

```diff
@@ -0,0 +1,305 @@
+import random
+import pytest
+import json
+import os
+import time
+from azext_iot.common.embedded_cli import EmbeddedCLI
+from azext_iot.tests.settings import DynamoSettings, ENV_SET_TEST_IOTHUB_REQUIRED, ENV_SET_TEST_IOTHUB_OPTIONAL
+from azure.cli.command_modules.iot.tests.latest._test_utils import _create_test_cert, _delete_test_cert
+from azext_iot.tests.iothub import IoTLiveScenarioTest
+from azext_iot.tests.test_constants import ResourceTypes
+from azext_iot.tests.generators import generate_generic_id
+from azext_iot.tests.helpers import add_test_tag
+from azext_iot.tests.iothub import DATAPLANE_AUTH_TYPES
+
+settings = DynamoSettings(req_env_set=ENV_SET_TEST_IOTHUB_REQUIRED, opt_env_set=ENV_SET_TEST_IOTHUB_OPTIONAL)
+CWD = os.path.dirname(os.path.abspath(__file__))
+
+CERT_FILE = "test_cert.cer"
+KEY_FILE = "test_key.cer"
+DATAPLANE_AUTH_TYPES.remove("cstring")
```

Comment:
Better to define your own auth types list rather than modify the DATAPLANE_AUTH_TYPES list. See this example https://github.com/msmoot0/azure-iot-cli-extension/blob/mirabai/azext_iot/tests/iothub/devices/test_iothub_device_tracing.py#L14